src/promnesia/__main__.py

```diff
@@ -309,19 +372,47 @@ def main() -> None:
         p.print_help(sys.stderr)
         sys.exit(1)
 
+    overwrite_policy_var = os.environ.get("PROMNESIA_INDEX_POLICY")
+    if overwrite_policy_var:
+        overwrite_policy_var = overwrite_policy_var.lower()
+        if overwrite_policy_var not in ("update", "overwrite"):
+            print(
```

Comment:
logger.error?

Reply:
Just copied code from [above](https://github.com/karlicoss/promnesia/blob/e44a2bda053cc93bb8dc1b84fd1a4461aa5d914f/src/promnesia/__main__.py#L308),
which didn't have logging.  Additionally, that's an do-it-once "static" error, notprobable to change between runs, so you would detect and fix it the 1st time it rings.
But i can add a log-statement, if you want it?

My personal pattern for errors that should _exit politely with some error-code and a message ("expected errors")_,
is to raise a new exception with those data, and i wrap all CLI code into error-handling+logging just for that exception type - all other exceptions dump stack-trace at the console ("unexpected errors").

I would prefer to submit a separate PR to substitute all such code places, if you are ok?

Reply:
Ah yes good point. Either way, it's a minor thing.

> My personal pattern for errors that should exit politely with some error-code and a message ("expected errors"),
is to raise a new exception with those data, and i wrap all CLI code into error-handling+logging just for that exception type - all other exceptions dump stack-trace at the console ("unexpected errors").

Yeah, this makes sense. I guess also makes it more convenient to test (so you can catch the exception). Not a big issue perhaps, but yeah if you have time for such PR -- sure!